application/views/survey/questions/answer/multiplenumeric/rows/sliders/answer_row.twig

```diff
@@ -39,7 +39,7 @@
                 <div class='col-xs-12 col-sm-2 slider-left text-right'>{{ sliderleft }}</div>
             {% endif %}
 
-            <div class="slider-container ls-input-group col-xs-12 col-sm-{{sliderWidth}}" style="min-height:15px;">
+            <div class="slider-container ls-input-group col-xs-12 col-sm-{{sliderWidth}}" style="min-height:15px;" id="answer{{myfname}}slid-container">
```

Comment:
`min-height:15px` , maybe we can include this in a css file ? It don't work without ?